src/query.ts

```diff
@@ -8,11 +8,22 @@ import { ChangeEvent } from './index';
  */
 export type QueryOptions = {
     filter: any,
+    options: FilterOptions,
     documents: any[],
     changes: Observable<ChangeEvent>,
 };
 
 /**
+ * FilterOptions
```

Comment:
After merging options, we'll only be using this from `collection.find` method. So we can move this to the `collection.ts` file. This is a public type so export it 

```diff
-export { Collection } from './collection';
+export { Collection, FindOptions } from './collection';
```